rewrite-xml/src/main/java/org/openrewrite/xml/ChangeNamespaceValue.java

```diff
@@ -41,94 +46,93 @@ public String getDescription() {
         return "Alters XML Attribute value within specified element of a specific resource versions.";
     }
 
+    @Nullable
     @Option(displayName = "Element name",
             description = "The name of the element whose attribute's value is to be changed. Interpreted as an XPath Expression.",
             example = "property")
```

Comment:
`required` defaults to `true`, so if this is truly nullable then we likely need to add that flag to the annotation here and elsewhere.
```suggestion
            example = "property",
            required = false)
```